collector/library/pod.js

```diff
@@ -46,7 +49,7 @@ class Pod {
 
         this.serviceName = serviceName;
         this.targetUrl = targetUrl;
-        this.checkInterval = checkInterval;
+        this.checkIntervalInMilli = checkInterval;
```

Comment:
We basically don't want to hammer logstash, not drain our network connections and basically provide some kind of spacing between the other pods competing for resources. It can maybe be lowered but I do think it makes sense in having it. Since we "retry" for various and diverse reasons - it makes a lot of sense that it's here.
The only case, in my opinion where it's not really necessary is when we successfully finished inserting a batch and we can easily move to the next batch. In all other cases of errors, you'd want to allow spacing between attempts and let the other systems (logstash and the logs server) some breathing room. That is my rubber ducking to why I think it's needed here.